back/tools/install-mariadb

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+set -e
+
+ostype="$([[ "${OSTYPE}" == "darwin"* ]] && echo "darwin" || echo "linux")"
+
+if [[ ! $(hash mysql 2>/dev/null) ]]; then
+	if [[ "${ostype}" == "linux" ]]; then
+		sudo apt update
+		sudo apt install software-properties-common
+
+		keysrv="hkp://keyserver.ubuntu.com:80"
+		sudo apt-key adv --recv-keys --keyserver "${keysrv}" 0xF1656F24C74CD1D8
```

Comment:
Unfortunately, this line doesn't work with WSL -ubuntu 10.4. 
My work around was found in [this issue](https://github.com/Microsoft/WSL/issues/3286#issuecomment-402631615)

Reply:
I'd prefer to make it seamless for WSL users, if possible. Is this something you can add? If not, would you be willing/able to simplify the corrections I need to make by testing the suggested alternative and providing a mini-script that I can refer to?